### DISCUSSION TITLE
[Feature] filter also by TAG

### DISCUSSION THREAD

**Discussion by matheuseduardo** [damongolding/immich-kiosk#209] [Feature request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Just a suggestion of a new filter, by "tag", should be really useful. I think would be really easy to – maybe just – tag some pictures that you want to display.

**Comment by damongolding** [comment]
That's a lovely suggestion!

I haven't used tags in Immich really, I'll do some research 🧐.

**Comment by damongolding** [comment]
Unfortunately the api ain't really set up for search by tag 😭. A Immich dev said it's on the todo list though.

**Comment by damongolding** [comment]
This will be added in 0.17.0
